frontend/src/main/pages/PlayPage.js

```diff
@@ -47,6 +47,20 @@ export default function PlayPage() {
     );
```

Comment:
We don't need another backend GET for normal commons; you can use `commonsPlus.commons` instead, avoiding an unnecessary query